src/engine/basics/tileData.hxx

```diff
@@ -53,7 +53,8 @@ struct TileData
   std::string author;
   bool drawGround = false;   /// draw ground below sprite (grass, flowers, ...)
   bool placeOnGround = true; /// wether or not this building is placeable on ground
-  bool placeOnWater = false; /// whether or not this building is placeable on ground
+  bool placeOnWater = false; /// whether or not this building is placeable on water
+  bool isOverPlacable;
```

Comment:
If you want to keep `isOverPlacable` as a bool, then in the JSON you should be constistent. If there's a reason to keep it as an int, then you should also make this an int

Reply:
np, will do that in the next PR